synchronization/utils.py

```diff
@@ -24,4 +22,24 @@ def get_salck_api_emails(data):
         if "email" in profile:
             users_emails.append(profile["email"])
             
-    return users_emails
\ No newline at end of file
+    return users_emails
+    
+    
+def get_salck_api_group_id(data):
```

Comment:
You misspelled slack in the method definition. This happened in other places, so make sure to fix all of them.